editors/shared/components/forms/SinglePhIdForm.tsx

```diff
@@ -1,9 +1,11 @@
+import { Skeleton } from "../ui/skeleton.js";
 import { GenericPHIDForm } from "./generics/GenericPHIDForm.js";
 import type { Maybe } from "document-model";
 import type { PHIDOption } from "@powerhousedao/document-engineering/ui";
 import type React from "react";
 
 interface SinglePhIdFormProps {
+  loading: boolean;
```

Comment:
Why loading  the loading prop is required.

Is  there a specific reason why loading needs to be mandatory? I was wondering if it might make more sense to make it optional, especially in cases where the component might not need to handle loading states. What do you think or do we need it that way ??